meson.build

```diff
@@ -31,6 +34,10 @@ config_h.set_quoted('LOCALEDIR', join_paths(get_option('prefix'),
                                  get_option('datadir'), 'locale'))
 config_h.set10('ENABLE_NLS', true)
 
+if host_machine.system() == 'darwin'
+  config_h.set('HAVE_GTK_MAC', true)
```

Comment:
Could make this actually optional? `required: false` above and set value to `libgtk_mac_integration_dep.found()`